unit-tests/algo/d2rgb/scene-data.h

```diff
@@ -12,7 +12,7 @@
 
 inline std::string bin_dir( std::string const & scene_dir )
 {
-    return join( join( scene_dir, "binFiles" ), "ac2" );
+    return join( join( scene_dir, "binFiles\\" ), "ac2\\" );
```

Comment:
No no no -- why? :)
That's the reason there's a join()!

Reply:
done